addons/last-edit-tooltip/addon.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Shared/edited dates tooltip",
-  "description": "Hover over a project's share date for information on when exactly it was shared and last edited.",
+  "description": "Hover over a project's share date for information on when exactly it was shared, last edited and created.",
```

Grammatical suggestion:
```suggestion
  "description": "Hover over a project's share date for information on when exactly it was created, shared, and last edited.",
```
Putting "created" first might make more sense as it's the first thing you do to start a Scratch project. Also, if "last edited", along with the word "last", is the final thing said, that makes it clear that the only thing that the word "last" applies to is "edited".

---

Could this order be applied to the tooltip as well?

---

Yes, I'm just about to commit it.